### DISCUSSION TITLE
error while saving a JS isostring date to mysql

### DISCUSSION THREAD

**Discussion by Nithur-M** [sidorares/node-mysql2#1869] [Q&A]
I was using mysql2 version 2.3.3 and it saved the date as it was. Today I updated the package to the latest version and this error pops up:
`Incorrect datetime value: '2023-02-26T18:09:56.000Z' for column 'pubDate' at row 1 (errno 1292) (sqlstate 22007) `
Also note that I moved the hosting from RDS to planetscale. Might I have configured the MySQL differently in the latest server?
Please help. Thanks.

**Comment by sidorares** [comment]
looks like a server error so solution might be outside of the driver

Can you show your table schema and an example code that inserts the date value?

**Comment by Nithur-M** [comment]
Here is the schema corresponding to the pubDate:
```
tags | json
pubDate | datetime
link | varchar(250)
```
example code that inserts the data:
`const [rows, fields] = await con.query('INSERT INTO jobs SET ?', [job]);`
Here is the corresponding job object:
`const job = {..., pubDate: new Date(item.date).toISOString(), link: item.url, savedDate: new Date(), ...};`

Thank you.

**Comment by Nithur-M** [comment]
I just noticed that `character_set_system` in the old DB is `utf8mb3` and in the new DB, it is `utf8`. Would it be the reason for this issue?
I am researching how to change character_set_system now. :p

**Reply by sidorares** [reply]
In mysql utf8 is an alias for utf8mb3
